src/x_vexp_fun.c

```diff
@@ -537,14 +537,15 @@ ex_toint(t_expr *e, long int argc, struct ex_ex *argv, struct ex_ex *optr)
                 FUNC_EVAL_UNARY(left, toint, (int), optr, 0);
         }
 
-#ifdef _MSC_VER
-/* rint is now advertised as part of the microsoft SDK but my MSVC still
-doesn't find it - so here it is again. */
+#ifdef __MSC__
```

Comment:
hmm, i cannot find the `__MSC__` macro on https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros (which i think is a canonical list).
probably this should just be replaced with `#ifdef _MSC_VER`

Reply:
> probably this should just be replaced with `#ifdef _MSC_VER`

Definitely!

Reply:
Good point! Thanks you.